lib/licenses.nix

```
@@ -1212,6 +1212,12 @@ lib.mapAttrs mkLicense ({
     fullName = "TCL/TK License";
   };
 
+  tost = {
```

Where does this name come from?

If you made it up, I think a direct acronym would be better.

```suggestion
  tostl = {
```